UOP1_Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.11f1
-m_EditorVersionWithRevision: 2019.4.11f1 (2d9804dddde7)
+m_EditorVersion: 2019.4.13f1
```

Comment:
Should not be pushed with your changes. See #96.

Reply:
Ok.